### ISSUE TITLE
Get involved card component

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#143]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue is to create a component within `Cards` that will present information about getting involved in an organization or event. Designs for this element can be found in the [Figma designs](https://www.figma.com/file/I9McFfaLu1RiiWp5IP3YjE/activist_public_designs?type=design&node-id=766%3A214&t=LzDq2xXwxoNIyVmo-1) and can also be seen below:

![Join](https://github.com/activist-org/activist/assets/24387426/31b129f5-6c6d-4848-827b-8536612d8781)

The organizations version would thus have feed cards from #142. For events it would look like this (no feed cards):

![Description Section](https://github.com/activist-org/activist/assets/24387426/dd560b7b-1d22-4c3a-9d67-079a3c4fede8)

### Contribution

Would be happy to get some help with this!

**Comment by momanyisamuel** [comment]
Working on this @andrewtavis please let me know on the changes so far

**Comment by andrewtavis** [comment]
Will check it tomorrow morning, @momanyisamuel 😊 Assigned and you have my thanks!

**Comment by momanyisamuel** [comment]
While the outer card style is similar the content seems different may I propose we have two different card components for get involved feeds and get involved events??

**Comment by andrewtavis** [comment]
I was thinking that the feed would go within the [Connect component](https://github.com/activist-org/activist/blob/main/frontend/components/Cards/CardsConnect.vue) :) Does that make sense? We give people the option to add in a feed, have their socials from which the feed would come, both or neither?

If so then the get involved one would be more focused on have a join CTA and would itself be a separate component as you suggested 😊

**Comment by andrewtavis** [comment]
Difference for this component would be more is it an event or is it an organization, and then that would determine if there’s the working group cards under for orgs or if there are the legal rights and other information for events :) We can also do two different ones for orgs and events though 🤔 No stress 😎

**Comment by momanyisamuel** [comment]
okay let me make some changes based on this

**Comment by andrewtavis** [comment]
Sounds great! Looking forward, @momanyisamuel! I'll start going through and merging some stuff now, but will wait on the changes before merging #145 :)

**Comment by momanyisamuel** [comment]
This should have both feed and event options the props can be changed respectively let me know if this will work

**Comment by andrewtavis** [comment]
Merged #145 just now, @momanyisamuel. Will edit it all a bit and then merge this when it's all done 😊

**Comment by andrewtavis** [comment]
Lots of changes went through to close this in d33d488, @momanyisamuel :) Decided to split it up into a feed and a feed item, and also added lots of type classes so that we can get ready for the backend. They're generally similar to what was there before, but more of a focus on inputting the data from globally defined classes in [frontend/types](https://github.com/activist-org/activist/tree/main/frontend/types) (new from Code Night). Current versions are at:

- https://activist.org/organizations/tech-from-below/
- https://activist.org/events/demo-at-brandenburg-gate/

Closing this 😊 Thanks for the help! I'll be making more issues soon, and I'm generally thinking that the backend will get going in a few weeks.
